flink-runtime/src/test/java/org/apache/flink/runtime/execution/librarycache/FlinkUserCodeClassLoadersTest.java

```diff
@@ -208,6 +212,50 @@ public void testRepeatedParentFirstPatternClass() throws Exception {
         childClassLoader.close();
     }
 
+    @Test
+    public void testGetClassLoaderInfo() throws Exception {
+        final ClassLoader parentClassLoader = getClass().getClassLoader();
+
+        final URL childCodePath = getClass().getProtectionDomain().getCodeSource().getLocation();
+
+        final URLClassLoader childClassLoader =
+                FlinkUserCodeClassLoaders.childFirst(
+                        new URL[] {childCodePath},
+                        parentClassLoader,
+                        new String[] {},
+                        NOOP_EXCEPTION_HANDLER,
+                        true);
+
+        String formattedURL = ClassLoaderUtil.formatURL(childCodePath);
+
+        assertEquals(
+                ClassLoaderUtil.getUserCodeClassLoaderInfo(childClassLoader),
+                "URL ClassLoader:" + formattedURL);
+
+        childClassLoader.close();
+    }
+
+    @Test
+    public void testGetClassLoaderInfoWithClassLoaderClosed() throws Exception {
+        final ClassLoader parentClassLoader = getClass().getClassLoader();
+
+        final URL childCodePath = getClass().getProtectionDomain().getCodeSource().getLocation();
+
+        final URLClassLoader childClassLoader =
+                FlinkUserCodeClassLoaders.childFirst(
+                        new URL[] {childCodePath},
+                        parentClassLoader,
+                        new String[] {},
+                        NOOP_EXCEPTION_HANDLER,
+                        true);
+
+        childClassLoader.close();
+
+        assertThat(
+                ClassLoaderUtil.getUserCodeClassLoaderInfo(childClassLoader),
+                is(startsWith("Cannot access classloader info due to an exception.")));
```

Comment:
```suggestion
                startsWith("Cannot access classloader info due to an exception."));
```
I would think that `is` isn't necessary.